ubuntu1704-desktop.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_comment": "Build with `packer build -var-file=ubuntu1704-desktop.json ubuntu.json`",
+  "vm_name": "ubuntu1704-desktop",
+  "desktop": "true",
+  "locale": "en_US.UTF-8",
+  "cpus": "1",
+  "disk_size": "130048",
```

Comment:
It's the size of the hard_disc (VM). 
The discussion you did start is a discussion regarding an open pull request. So please do not comment the pull request as long as the topic isn't the pull request.